src/pages/dashboard/[subdomain]/editor.tsx

```diff
@@ -683,7 +684,7 @@ export default function SubdomainEditor() {
                   <div className="mt-5 flex-1 flex overflow-hidden">
                     {isMobileLayout ? (
                       !isRendering ? (
-                        <Editor
+                        <CodeMirrorEditor
                           value={values.content}
```

Comment:
I just noticed that when I fixed the cursor issue earlier, I overlooked the editor on mobile. Since it's a small change, I hope you can patch it directly in this PR. Thank you very much.

Reply:
Just need to change `values.content` to `initialContent`